docs/api/catalog-resource.md

```diff
@@ -235,6 +235,7 @@ summary                 | string                     | no       |
 tags                    | array of strings           | no       |
 title                   | string                     | no       |
 verbatimVersion         | string                     | no       | The version string as it's originally found in the .nuspec
+vulnerabilities         | array of objects           | no       | The security vulnerabilities of the package
```

Comment:
I'm not an expert on the server implementation, but isn't "catalog" intended to be append only?  If a package is brand new, uploaded to nuget.org for the first time, how could it already have a known vulnerability?

Reply:
An existing package can appear (for a second time or Nth) in the catalog. When a vulnerability is associated with a package it will appear in the catalog a little after that association.

Reply:
We can create multiple catalog leaves for the same package. The package details leaf captures a snapshot of the package's metadata at that point in time. For example:

1. Package FooBar is uploaded. We create a new catalog leaf for this package. It has no vulnerabilities so its array is empty
1. A vulnerability in "FooBar" is found. We discover this using GitHub's API
1. We create a new catalog leaf for FooBar, this time, the vulnerabilities array contains the vulnerability information.

Reply:
This is the same kind of thing that happens when a package is unlisted. It appeared at time X originally as listed then it appears at time Y (Y > X) as unlisted. All of the metadata is the same except a timestamp and the listed flag.